android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -149,6 +150,7 @@ public synchronized void onServiceConnected(ComponentName className, IBinder ser
         mDiscoveryService.restartScan();
       }
       mPwCollection = mDiscoveryService.getPwCollection();
+      onUrlDeviceDiscoveryUpdate();
```

Comment:
It looks like you included part of Jordan's bugfix here, could you remove it so he can land it separately?  Or is it also needed for your change?